Leanplum-SDK/Classes/Internal/Leanplum.m

```diff
@@ -1644,6 +1644,59 @@ + (void)handleNotification:(NSDictionary *)userInfo
     [[LPCountAggregator sharedAggregator] incrementCount:@"handle_notification"];
 }
 
```

Comment:
Is this PR incomplete? I do not see the code where we check if swizzling should be enabled or disabled. Also, how does it help by moving all the code to leanplum.m?

Reply:
Yes the PR is complete functionality wise. I still need to fix tests and add see if i can add test cases.